src/screens/OnBoardingProfileScreen.tsx

```diff
@@ -1,19 +1,132 @@
 import { NativeStackScreenProps } from "@react-navigation/native-stack";
-import { Box, Button, Text } from "native-base";
+import { Box, Button, Text, Heading, FormControl, Input, Radio, VStack } from "native-base";
 import { FC } from "react";
+import { useForm, Controller, Resolver } from "react-hook-form";
 
+import { signUp } from "../actions/user.actions";
+import { useAppDispatch } from "../hooks/redux";
 import { RootStackParamList } from "../navigators/RootNavigator";
+import Logo from "./../components/Logo";
+
+const OnBoardingProfileScreen: FC<NativeStackScreenProps<RootStackParamList>> = () => {
+  const dispatch = useAppDispatch();
+
+  const resolver: Resolver<FormValues> = async (values) => {
```

Comment:
As a tip, since this function does not depend on any state, it could be moved outside the component. This will allow the function to not be recreated every time the component changes